test/spec/video_spec.js

```diff
@@ -1,48 +1,67 @@
 import { isValidVideoBid } from 'src/video';
-const utils = require('src/utils');
+import { auctionManager } from 'src/auctionManager';
 
 describe('video.js', () => {
   afterEach(() => {
-    utils.getBidRequest.restore();
+	  auctionManager.getBidsRequested.restore();
   });
 
   it('validates valid instream bids', () => {
-    sinon.stub(utils, 'getBidRequest', () => ({
-      bidder: 'appnexusAst',
-      mediaTypes: {
-        video: { context: 'instream' },
-      },
-    }));
+	  sinon.stub(auctionManager, 'getBidsRequested', () => [
```

Comment:
Add these stubs in `beforeEach`.  And use `stub.returns` method in each test case. 
This will prevent error `sinon stub already wrapped` if any test case fails and reduce lines of code too.